hnginternship5/ziki
# Changed Index.html to use CSS variables in a separate CSS file

### PURPOSE OF PR
Allows homepage to use CSS variables for theming


### FEATURES IMPLEMENTED
Moved CSS to separate file so that it can be built using gulp
Added CSS variables to enable theming

### HOW TO TEST

Open Homepage to test for dark mode and see that it works because of the CSS variables introduced.
